projects/rocprofiler-systems/source/bin/rocprof-sys-sample/impl.cpp

```diff
@@ -373,11 +373,19 @@ parse_args(int argc, char** argv, std::vector<char*>& _env)
             if(_v.size() > 1) update_env(_env, "ROCPROFSYS_OUTPUT_PREFIX", _v.at(1));
         });
     parser
-        .add_argument({ "-T", "--trace" }, "Generate a detailed trace (perfetto output)")
+        .add_argument({ "-T", "--trace" },
+                      "Generate a detailed trace (perfetto output) from cached data")
         .max_count(1)
         .action([&](parser_t& p) {
             update_env(_env, "ROCPROFSYS_TRACE", p.get<bool>("trace"));
         });
+    parser
+        .add_argument({ "-L", "--trace-legacy" },
+                      "Generate a detailed trace (perfetto output)")
```

Comment:
We should change the help message here too.
```suggestion
                      "Generate a detailed trace (perfetto output) using the legacy method")
```